resources/sass/app.scss

```diff
@@ -239,86 +237,86 @@ dl.import-meta .download-csv {
 }
 
 .button {
```

Comment:
All the replacements appear to be correct, and yet the focus and active styles of the buttons in the store that use this class are off. Could it be related to the order of the pseudo-classes?

Reply:
An unwanted outline is applied to active and focused buttons in the store. To avoid it, we could use `outline: none`